pkg/k8s/apis/spiderpool.spidernet.io/v2beta1/spidermultus_types.go

```diff
@@ -95,6 +99,10 @@ type SpiderIPvlanCniConfig struct {
 	// +kubebuilder:validation:Optional
 	Bond *BondConfig `json:"bond,omitempty"`
 
+	// +kubebuilder:default=false
+	// +kubebuilder:validation:Optional
+	EnableRdma bool `json:"enableRdma"`
```

Comment:
I am wondering whether there should be a new field called "rdmaResourceName" here

Reply:
done, maybe we can only use one field for this. If the value is empty, which means we don't use the rdma plugin. WDYT?